readthedocs/search/documents.py

```diff
@@ -60,14 +61,19 @@ class Meta(object):
     content = fields.TextField(attr='processed_json.content')
     path = fields.TextField(attr='processed_json.path')
 
+    # Fields to perform search with weight
+    search_fields = ['title^10', 'headers^5', 'content']
```

Comment:
Is this used by the `DocType` class directly, or are we storing it here to add later? If the prior, we should probably set it in an `__init__` method or somewhere else, so it doesn't get confused with the other data here.

Reply:
Its not actually used by `DocType` class, but we are using it in [class method](https://github.com/rtfd/readthedocs.org/pull/4309/files#diff-7faf5e6b86776ca9b80505230b669285R106). I dont know if keeping it in `__init__` method will make it work in class method!

Reply:
Hrm, good point. I guess it's fine for now then.